webapp/channels/src/components/advanced_create_post/priority_labels.tsx

```diff
@@ -97,6 +94,21 @@ function PriorityLabels({
     priority,
     requestedAck,
 }: Props) {
+    const messages = defineMessages({
```

Comment:
please use `useIntl` hook directly here as defineMessages usage is usually used in places where its difficult to get access to intl prop